layouts/community/ergodox/meagerfindings/keymap.c

```diff
@@ -0,0 +1,566 @@
+  #include QMK_KEYBOARD_H
+#include "debug.h"
```

Comment:
Could you removal all but the `QMK_KEYBOARD_H` and the version.h includes? 

The rest should be included already.

Reply:
Removed